.github/workflows/dev-ci.yml

```diff
@@ -0,0 +1,375 @@
+name: Dev CI
+
+on:
+  workflow_dispatch:
```

Comment:
my understanding is that we have to wait that the PR is merged...

meanwhile it can be triggered by [GitHub CLI](https://cli.github.com/) using:

`gh workflow run --ref=gh-workflows`

See https://stackoverflow.com/a/67840292 and its most upvoted comment explaining the trick which I applied for Dev CI.

I re-ran it here: https://github.com/powsybl/powsybl-metrix/actions/runs/11706681139